### DISCUSSION TITLE
Create windows service for strapi

### DISCUSSION THREAD

**Discussion by pvk2410** [strapi/strapi#7722] [Legacy Topics]
Hello Community members,

I have a strapi v3.1.4 running on window servers and currently the only way i start it up is with the command 'npx strapi develop' (in my dev instance) from command prompt.

Is there a way to register the windows service agent for this instance? If possible, how can it be configured to start in the develop mode ?

Regards.

**Comment by derrickmehaffy** [comment]
Not natively with us @pvk2410 you will need to look into something like pm2. Strapi ourselves have no service manager.
